From 02d021c900454d494a8b38a24953b9bb18da702c Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Wed, 16 Nov 2016 18:56:38 +0200
Subject: [PATCH 171/278] drivers/tty/serial: Remove a Bringup Change

During Axxia bringup, setting the baud rate was disabled because
the frequency reported for the peripheral clock was wrong.

This commits reverts that change.

Signed-off-by: John Jacques <john.jacques@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 drivers/tty/serial/amba-pl011.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/tty/serial/amba-pl011.c b/drivers/tty/serial/amba-pl011.c
index c1b72dd..6b6fd82 100644
--- a/drivers/tty/serial/amba-pl011.c
+++ b/drivers/tty/serial/amba-pl011.c
@@ -2061,10 +2061,8 @@ static void sbsa_uart_shutdown(struct uart_port *port)
 			quot -= 2;
 	}
 	/* Set baud rate */
-#if 0
 	pl011_write(quot & 0x3f, uap, REG_FBRD);
 	pl011_write(quot >> 6, uap, REG_IBRD);
-#endif
 
 	/*
 	 * ----------v----------v----------v----------v-----
-- 
1.8.5.2.233.g932f7e4

